docs/source/configure.rst

```diff
@@ -80,24 +80,24 @@ The ``qpc server config`` command takes the following options:
 - The ``--host`` option is required. If you are using the qpc command line tool on the same system where the server is running, you can supply the loopback address ``127.0.0.1`` as the value. Otherwise, supply the IP address for your Quipucords server.
 - The ``--port`` option is optional. The default value for this option is ``443``. If you decided to remap the Quipucords default exposed server port to another port, the port option is required. You must supply the port option and the remapped value in the command, for example, ``--port 8443``.
 
-For example, if you are configuring the command line tool on the same system as the server and the server uses the default exposed server port, you would enter the following command to configure the qpc command line tool:
+For example, if you are configuring the command line tool on the same system as the server and the server uses the default exposed server port, you would enter the following command to configure the qpc command line tool::
 
-  ``# qpc server config --host 127.0.0.1``
+  # qpc server config --host 127.0.0.1
 
-However, if you are configuring the command line tool on a system that is remote from the server, the Quipucords server is running on the IP address 192.0.2.0, and the port is remapped to 8443, you would enter the following command to configure the qpc command line tool:
+However, if you are configuring the command line tool on a system that is remote from the server, the Quipucords server is running on the IP address 192.0.2.0, and the port is remapped to 8443, you would enter the following command to configure the qpc command line tool::
 
-  ``# qpc server config --host 192.0.2.0 --port 8443``
+  # qpc server config --host 192.0.2.0 --port 8443
 
 .. _login:
 
-Logging in to and Logging out of the qpc Command Line Interface
----------------------------------------------------------------
-After the connection between the qpc command line tool and the Quipcords server is configured on the system where you want to use the qpc command line interface, you can log in to the interface and begin using it to run qpc commands.
+Logging in to the Quipucords Server
+-----------------------------------
+After the connection between the qpc command line tool and the Quipucords server is configured on the system where you want to use the command line interface, you can log into the server and begin using the command line interface to run qpc commands.
```

Comment:
Grammar is not my strong point but we use both `log in to` and `log into` --> not sure if we want to change this to be consistent or if it doesn't make a difference.

Reply:
These are Jamie's edits that I'm porting forward so I leave the word smithing to her.